src/openforms/submissions/api/validators.py

```diff
@@ -57,6 +61,47 @@ def __call__(self, data: dict, field: JSONField):
             raise serializers.ValidationError(errors)
 
 
+class ValidateFamilyMembersPrefillData:
+    code = "invalidFamilyMembersPrefilledField"
+    default_message = _("The family members prefill data may not be altered.")
+    requires_context = True
+
+    def __call__(self, submitted_data: dict, field: JSONField):
+        instance: SubmissionStep = field.parent.instance
+        submission = instance.submission
+        prefill_data = submission.get_prefilled_data()
```

Comment:
now that I see the actual validation code and what is needed to validate - I have the impression that we *can* move this to the component itself, and then we don't have to violate the contrib/plugin-specific imports in a core module :thinking: